src/Controls/src/Core/Routing.cs

```diff
@@ -126,7 +126,7 @@ internal static string[] GetRouteKeys()
 			return keys;
 		}
 
-		public static Element GetOrCreateContent(string route)
+		public static Element GetOrCreateContent(string route, IServiceProvider services)
```

Comment:
Can we maintain the method that takes one parameter as well? that way we don't break people that may be using this API?